File: Python/ScrumMaster.py
#!/usr/bin/python
# -*- coding: UTF-8 -*-
import Equipo
from typing import List

class ScrumMaster(object):
	def getNombre(self) -> String:
		return self.__nombre

	def setNombre(self, aNombre : String) -> None:
		self.__nombre = aNombre

	def getRolSecundario(self) -> String:
		return self.__rolSecundario

	def setRolSecundario(self, aRolSecundario : String) -> None:
		self.__rolSecundario = aRolSecundario

	def ScrumMaster(self, aNombre : String, aRolSecundario : String):
		pass

	def __init__(self):
		self.__nombre : String = None
		self.__rolSecundario : String = None
		self.hasA : Equipo = None
		"""# @AssociationMultiplicity 1"""

File: Java/ScrumMaster.java
package scrum;

public class ScrumMaster {

    private String nombre;
    private String rolSecundario;

    public ScrumMaster(String nombre) {
        this.nombre = nombre;
        this.rolSecundario = "";
    }
    
    public String getNombre() {
        return this.nombre;
    }

    public void setNombre(String aNombre) {
        this.nombre = aNombre;
    }

    public String getRolSecundario() {
        return this.rolSecundario;
    }

    public void setRolSecundario(String aRolSecundario) {
        this.rolSecundario = aRolSecundario;
    }

    @Override
    public String toString() {
        StringBuilder sb = new StringBuilder();
        sb.append("ScrumMaster{");
        sb.append("nombre=").append(nombre);
        sb.append(", rolSecundario=").append(rolSecundario);
        sb.append('}');
        return sb.toString();
    }

}
